Resources/public/js/views/ez-languageswitcherview.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageswitcherview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Language Switcher View class
+     *
+     * @module ez-languageswitcherview
+     */
+    Y.namespace('eZ');
+
+    var EXPANDED_LANGUAGE_LIST = 'is-expanded';
+
+    /**
+     * The raw content view
+     *
+     * @namespace eZ
+     * @class LanguageSwitcherView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSwitcherView = Y.Base.create('languageSwitcherView', Y.eZ.TemplateBasedView, [Y.eZ.AccordionElement], {
+        events: {
+            '.ez-selected-language': {
+                'tap': '_expandList'
+            },
+            '.ez-content-translation a': {
+                'tap': '_switchLanguage'
+            }
+        },
+
+        initializer: function () {
+        },
+
+        /**
+         * Renders the language switcher view
+         *
+         * @method render
+         * @return {eZ.LanguageSwitcherView} the view itself
+         */
+        render: function () {
+            var container = this.get('container');
+
+            container.setHTML(this.template({
+                content: this.get('content').toJSON(),
+                //todo: get current translation and pass it here
```

Comment:
the current language is the one in the URI (currently added in https://github.com/ezsystems/PlatformUIBundle/pull/233) so it should probably be passed from the LocationViewViewService to the LocationView and then passed to the LanguageSwitcherView.

@yannickroger in https://github.com/ezsystems/PlatformUIBundle/pull/233, can you add the `languageCode` to the object returned by `getViewParameters` and add the corresponding attribute the LocationView.

@mhyndle for now, you can add the languageCode attribute to the locationView and hardcode it.